pyccl/neutrinos.py

```diff
@@ -25,41 +32,49 @@ def Omeganuh2(a, Neff, mnu, TCMB=2.725):
     # Convert to array if it's not already an array
     if not isinstance(a, np.ndarray):
         a = np.array([a,]).flatten()
+    if not isinstance(mnu, np.ndarray):
+        mnu = np.array([mnu,]).flatten()
+    
+    # FIXME: Implement length checking of mnu (should it be a certain length?)
```

Comment:
FIXME here.

Reply:
See above. I am pretty sure (?) I didn't write this FIXME, but maybe I did. In any case, it's fixed now.